recipes/renode-cores/helpers/renode_build_with_cmake.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+CONFIGURATION="Release"
+HOST_ARCH="i386"
+CMAKE_COMMON=""
```

Comment:
See my previous comments about not relying so much on variables and simplifying logic - I suspect as with the other helper scripts we probably don't actually need separate scripts if we simplify this enough.